CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,9 @@
+# Debezium Code of Conduct
+
+We are committed to making partition in the Commonhaus Foundation a harassment-free experience for everyone, based on respect and ethical conduct.
+For the complete Code of Conduct, see the [Commonhaus Foundation Code of Conduct](https://github.com/commonhaus/foundation/blob/main/policies/code-of-conduct.md).
+
+## Debezium Contact Information
+
+* To report an issue, Code of Conduct reports should be sent to `report@DEBEZIUM-mailing-list`. We wil follow [CF CoC procedures](https://www.commonhaus.org/policies/code-of-conduct/#handling-reports-and-escalations) when handling Code of Conduct reports.
```

Comment:
Need to sync with CF to determine if this ML is one managed on the CF domain or whether this is one to be managed directly by the project.